server/app/services/applicant/ApplicantData.java

```diff
@@ -149,4 +149,15 @@ public ImmutableMap<Path, String> getFailedUpdates() {
   public boolean updateDidFailAt(Path path) {
     return getFailedUpdates().containsKey(path);
   }
+
+  public Optional<String> getDOB() {
+    String dob = readString(WellKnownPaths.APPLICANT_DOB).get();
+    return Optional.of(dob);
+  }
+
+  public void setDOB(@Nullable String dob) {
```

Comment:
suggestion (blocking): avoid `@Nullable` and prefer `Optional.ofNullable` instead